front-end/src/components/FriendRequestListModal.tsx

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { Button, Card, CardBody, CardTitle, Modal, ModalBody, ModalHeader } from 'reactstrap';
+import { Follow } from '../types/Follow';
+import FriendRequestItem from './FriendRequestItem';
+
+const FriendRequestListModal = (props: any) => {
```

Comment:
a props interface would be nice here just so that its clear what props are being passed in

Reply:
I tried that at first, but it gets really messy with having to deal with the `UserLogin` object being defined or undefined. I imagine there might be a similar issue in `CreatePost.tsx` which does the same thing since it also just passes in `props: any`